code/modules/research/designs/weapons.dm

```diff
@@ -163,6 +173,8 @@
 	materials = list(MAT_IRON = 5000, MAT_GLASS = 1000)
 	category = "Weapons"
 	build_path = /obj/item/weapon/gun/syringe/rapidsyringe
+	locked = TRUE
+	req_lock_access = list(access_medical, access_cmo)
```

Comment:
Can't imagine a situation in which someone would have CMO access but not medical.